### ISSUE TITLE
[Bug] PCSX2 Broken Parameters (nogui, fullscreen)

### ISSUE THREAD

**Issue by OdinVex** [dragoonDorise/EmuDeck#584]
I started this out over at [SteamGridDB](https://github.com/SteamGridDB/steam-rom-manager/issues/433), but it may belong here.

Modern versions of PCSX2 don't support `--nogui` and `--fullscreen`. It's a real mess over there at PCSX2, there's a Musk-meets-Twitter-like regard for code. Removing both from parameters resulted in a no-start until I restarted the Steam Client (SteamDeck). They're migrating to Qt, wxWidgets *support* is considered legacy and so is the entire 'old GUI'.

The new Qt GUI is a clone of DuckStation. It uses the `-fullscreen` option instead of `--fullscreen`, tested.

In short, swap out `--nogui --fullscreen` for `-fullscreen`.

**Comment by Godsbane** [comment]
We've been using the modern QT version for many months.

I can confirm everything you say here is correct. and are what we use already.

I'm not sure why you think we aren't using the correct parameters?

https://github.com/dragoonDorise/EmuDeck/blob/1353de6585db5a2dfb87df3f1accc91611afb511/configs/steam-rom-manager/userData/userConfigurations.json#L2980

as you can see here, we are using the modern QT params, so i am closing this.

**Comment by Godsbane** [comment]
Sounds like you never let steam rom manager uodate with emudeck.

We've been using the official qtappimage for quite a long time now.

**Comment by Godsbane** [comment]
the mmc path change was reverted by valve a couple hours after they released the original patch.

please upload your log file from your last Emudeck update. ~/emudeck/emudeck.log
